backend/core/views.py

```
@@ -5021,3 +5049,28 @@
+    def status(self, request):
+        return Response(dict(TaskNode.TASK_STATUS_CHOICES))
+
+    @action(
+        detail=False,
+        name="Get tasks for the calendar",
+        url_path="calendar/(?P<start>.+)/(?P<end>.+)",
+    )
+    def calendar(
+        self,
+        request,
+        start=timezone.now().date(),
```

Ruff (0.8.2)

Do not perform function call in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)